mods/ra/maps/allies-05a/allies05a.lua

```diff
@@ -152,7 +188,21 @@ FreeTanya = function()
 
 	Trigger.OnKilled(Tanya, function() ussr.MarkCompletedObjective(ussrObj) end)
 
-	KillSams = greece.AddPrimaryObjective("Destroy all four SAM Sites that block\nthe extraction helicopter.")
+	if Map.Difficulty == "Real tough guy" then
+		KillSams = greece.AddPrimaryObjective("Destroy all four SAM Sites that block\nour reinforcements' helicopter.")
+
+		greece.MarkCompletedObjective(mainObj)
+		surviveObj = greece.AddPrimaryObjective("Tanya must not die!")
+		Media.PlaySpeechNotification("TanyaRescued")
```

Comment:
`Fatal Lua Error: [string "BindingSupport.lua"]:30: Unable to convert parameter 0 to Player`